src/components/coursesSection/course/Course.module.css

```diff
@@ -0,0 +1,11 @@
+.course {
```

Comment:
it is better here to give the course and its image max-width for better rendering on the smaller screens.

Reply:
Done